src/rebar_git_resource.erl

```diff
@@ -282,3 +288,16 @@ parse_tags(Dir) ->
                     end
             end
     end.
+
+git_clone_options() ->
+    Option = case os:getenv("REBAR_GIT_CLONE_OPTIONS") of 
+        false -> "" ;       %% env var not set
+        [] -> "" ;          %% env var set to empty
```

Comment:
this clause would be covered fine under `Opt -> Opt`

Reply:
Sorry, I did not get your point. Could you explain it with more detail? 
 
Thx.

Reply:
`[] -> ""` basically means `"" -> ""` which the clause `Opt -> Opt` would catch anyway

Reply:
I see. 
[] and ""  render the same result for io:format("...~ts...").